model/UserSchema.js

```diff
@@ -20,10 +20,39 @@ const userSchema = new mongooose.Schema(
       type: String,
       required: true,
     },
+    tokens:[
+      {
+          token:{
```

Comment:
is prettier configured correctly in your editors?
indention changes will come and bite sometime later.

Reply:
All code files are now properly formatted using prettier.